core-bundle/src/Resources/contao/library/Contao/System.php

```diff
@@ -207,6 +212,11 @@ public static function importStatic($strClass, $strKey=null, $blnForce=false)
 		{
 			$container = static::getContainer();
 
+			if (null === $container)
+			{
+				throw new \RuntimeException('The Symfony container is not available, did you initialize the ContaoFramework?');
```

Comment:
```suggestion
				throw new \RuntimeException('The Symfony Container is not available, did you initialize the ContaoFramework?');
```

Reply:
Also, it should be "Contao Framework".

Reply:
Changed to "Contao framework" in 04378ab5492ef0e8418038e1e480f2a582db63fd

**Contao** and **Symfony** is a product name. **container** and **framework** is not. See
https://github.com/contao/contao/blob/master/core-bundle/src/Image/ImageFactory.php#L310
https://github.com/contao/test-case/blob/4.x/README.md#mocking-the-symfony-container
https://github.com/contao/test-case/blob/4.x/README.md#mocking-the-contao-framework
https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bundle/FrameworkBundle/Command/ContainerAwareCommand.php#L44